iron-dropdown.html

```diff
@@ -101,23 +193,17 @@
           },
 
           /**
-           * An animation config. If provided, this will be used to animate the
-           * opening of the dropdown. Pass an Array for multiple animations.
-           * See `neon-animation` documentation for more animation configuration
-           * details.
+           * The class defining the entry animation.
            */
-          openAnimationConfig: {
```

Comment:
this might break bindings done via html. We should probably keep these and warn if they get set, e.g.
```js
ready: function() {

  // ...
  if (this.openAnimationConfig || this.closeAnimationConfig || this.animationConfig) {
    console.warn('Configuring animations through animationConfig, ' +
                 'openAnimationConfig, or closeAnimationConfig is no longer ' +
                 'supported. Use entryAnimation and exitAnimation instead.');
    // Set this for backwards-compat.
    this.animationConfig = this.animationConfig || {
      open: this.openAnimationConfig,
      close: this.closeAnimationConfig
    };
  }
}
```

And update the description to something like
```
/**
 * Deprecated, use `entryAnimation` instead.
 * `iron-dropdown` doesn't depend anymore on `neon-animation`, and this property is kept
 * here to not break bindings. Setting it won't have effects on the animation.
 * @deprecated
 */